example/server/bin/docker-compose.yml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-version: '3'
+version: '2'
```

Comment:
any reason for downgrading format?

Reply:
My docker-compose doesn't work with version 3. And there seems to be no issues with version 2.